compiler/rustc_const_eval/src/interpret/machine.rs

```diff
@@ -215,14 +215,16 @@ pub trait Machine<'mir, 'tcx: 'mir>: Sized {
 
     /// Directly process an intrinsic without pushing a stack frame. It is the hook's
     /// responsibility to advance the instruction pointer as appropriate.
+    /// Returns an `Instance` in case the intrinsic was not handled, and a function should
+    /// be called instead.
```

Comment:
```suggestion
    ///
    /// Returns `None` if the intrinsic was fully handled.
    /// Otherwise, returns an `Instance` of the function that implements the intrinsic.
```